src/functions/template-tags/venue.php

```diff
@@ -159,7 +164,12 @@ function tribe_get_venue_label_plural() {
 		 *
 		 * @param string $label The plural version of the Venue label, defaults to "Venues" (uppercase)
 		 */
-		return apply_filters( 'tribe_venue_label_plural', esc_html__( 'Venues', 'the-events-calendar' ) );
+		return  esc_html(
```

Comment:
:no_entry_sign: Language constructs must be followed by a single space; expected 1 space but found "··"

<sub>hash: 5512a4b9ae247ae268bc65635914e777</sub>